source/_posts/2024-12-04-release-202412.markdown

```diff
@@ -407,6 +409,70 @@ update Home Assistant as usual, and you’re good to go.
 
 Easy, right? 😎
 
+## Patch releases
+
+We will also release patch releases for Home Assistant 2024.11 in November.
```

Comment:
2024.12 in December.

Reply:
Nice catch! Thanks 👍 

<img src="https://media1.giphy.com/media/1IPjwAJ0x5sGqB8nHn/giphy.gif"/>